File: test/fixtures/python/redirect_command_data.py
import subprocess
import requests

command1 = subprocess.run('echo myfile.jg', shell=True, capture_output=True, text=True).stdout

headers = {
    'Content-Type': 'application/x-www-form-urlencoded',
}

data = 'foo'
with open(command1) as f:
    data += '&' + f.read().replace('\n', '').replace('\r', '')

response = requests.post('http://localhost:28139', headers=headers, data=data.encode())

File: test/fixtures/r/redirect_command_data.r
library(httr)

headers = c(
  `Content-Type` = "application/x-www-form-urlencoded"
)

data = paste0("foo&@", system("echo myfile.jg", intern = TRUE))

res <- httr::POST(url = "http://localhost:28139", httr::add_headers(.headers=headers), body = data)
